src/HashBus.Projector.TopTweeters/App.cs

```diff
@@ -2,31 +2,45 @@ namespace HashBus.Projector.TopTweeters
 {
     using System.Collections.Generic;
     using System.Threading;
+    using System.Threading.Tasks;
+    using HashBus.Application.Events;
     using HashBus.NServiceBusConfiguration;
     using HashBus.ReadModel;
     using HashBus.ReadModel.MongoDB;
     using MongoDB.Driver;
     using NServiceBus;
+    using Tweet = HashBus.ReadModel.Tweet;
 
     class App
     {
-        public static void Run(string mongoConnectionString, string mongoDBDatabase, string endpointName)
+        public static async Task Run(string mongoConnectionString, string mongoDBDatabase, string endpointName, string applicationAddress)
         {
             var mongoDatabase = new MongoClient(mongoConnectionString).GetDatabase(mongoDBDatabase);
 
-            var busConfiguration = new BusConfiguration();
-            busConfiguration.EndpointName(endpointName);
-            busConfiguration.UseSerialization<JsonSerializer>();
-            busConfiguration.EnableInstallers();
-            busConfiguration.UsePersistence<InMemoryPersistence>();
-            busConfiguration.RegisterComponents(c =>
+            var endpointConfiguration = new EndpointConfiguration(endpointName);
+            endpointConfiguration.UseSerialization<JsonSerializer>();
+            endpointConfiguration.EnableInstallers();
+            endpointConfiguration.UsePersistence<InMemoryPersistence>();
+            endpointConfiguration.RegisterComponents(c =>
                 c.RegisterSingleton<IRepository<string, IEnumerable<Tweet>>>(
                     new MongoDBListRepository<Tweet>(mongoDatabase, "top_tweeters__tweets")));
-            busConfiguration.ApplyMessageConventions();
+            endpointConfiguration.ApplyMessageConventions();
+            endpointConfiguration.ApplyErrorAndAuditQueueSettings();
 
-            using (Bus.Create(busConfiguration).Start())
+            var transportExtensions = endpointConfiguration.UseTransport<MsmqTransport>();
+            var routing = transportExtensions.Routing();

```

Comment:
Please insert a blank line before this line (for consistency with other endpoints).